site/src/components/steps.tsx

```diff
@@ -28,22 +28,39 @@ const TutorialSteps: React.FC<TutorialStepProps> = (props) => {
             menuTitle
             order
           }
+          parent {
+            ... on File {
+              relativeDirectory
+            }
+          }
         }
       }
     }
   `);
 
-  const tutorialSteps = Object.entries(tutorialNodes).map(
-    ([key, tutorialNode]) => {
-      return (
-        <Step
-          key={key}
-          title={`Step ${tutorialNode.frontmatter.order}`}
-          description={tutorialNode.frontmatter.menuTitle}
-        />
-      );
-    },
-  );
+  const [tutorialType, setTutorialType] = useState('');
+
+  useEffect(() => {
+    setTutorialType(
+      window.location.pathname
+        .substr(0, window.location.pathname.lastIndexOf('/'))
+        .replace('/tutorials/', ''),
+    );
+  });
```

Comment:
```suggestion
  }, []);
```
Oops sorry about it. I was wrong on it. 😓